### PR TITLE
Instructions for validator logging

### PR DESCRIPTION
**Pull request by carllin**
#### Problem
Some validators' logs are being suppressed

#### Summary of Changes
Add instructions for disabling 

Fixes #

### CODE REVIEW

**Comment by t-nelson** [comment]
I think we can limit this to just the service running the validator process, rather than system-wide, with:
```text
[Service]
LogRateLimitIntervalSec=0
LogRateLimitBurst=0
```
```bash
systemctl daemon-reload
systemctl restart [solana-validator].service
```

**Comment by carllin** [comment]
@t-nelson is per unit journal configuration possible? I think it's still an open issue: https://github.com/systemd/systemd/issues/4751

**Comment by mvines** [comment]
I think we should advise not logging to syslog at all.   Instead log to a file and use logrotate to manage the logs, something like what we currently do for our nodes:
https://github.com/solana-labs/cluster/blob/545b1d7b2ae1125fb2b700ac66ffecfeebb3b286/bin/machine-setup.sh#L63-L73

**Comment by t-nelson** [comment]
> @t-nelson is per unit journal configuration possible? I think it's still an open issue: [systemd/systemd#4751](https://github.com/systemd/systemd/issues/4751)

```
       LogRateLimitIntervalSec=, LogRateLimitBurst=
           Configures the rate limiting that is applied to messages generated
           by this unit. If, in the time interval defined by
           LogRateLimitIntervalSec=, more messages than specified in
           LogRateLimitBurst= are logged by a service, all further messages
           within the interval are dropped until the interval is over. A
           message about the number of dropped messages is generated. The time
           specification for LogRateLimitIntervalSec= may be specified in the
           following units: "s", "min", "h", "ms", "us" (see systemd.time(7)
           for details). The default settings are set by RateLimitIntervalSec=
           and RateLimitBurst= configured in journald.conf(5).
```

That's from the manpage (`man systemd.exec`).  Sure sounds like the per-service equivalent.  Is that Issue not just for vacuum and disk size?

**Comment by carllin** [comment]
@mvines thanks for the suggestion
@t-nelson thanks for all the explanations!

File: docs/src/running-validator/validator-start.md [hunk 1]
```diff
@@ -196,6 +196,26 @@ solana-validator \
   --rpc-port 8899 \
   --entrypoint devnet.solana.com:8001 \
   --limit-ledger-size
+  --log ~/solana-validator.log
+```
+
+Lastly, to configure logging, please run the following:
```
Comment (t-nelson) [inline docs/src/running-validator/validator-start.md:202]:
```suggestion
Lastly, to configure log rotation, please run the following:
```

**Pull request merged**